templates/ai/data-science.html

```diff
@@ -272,7 +272,7 @@ <h2>Get leading open source ML tools seamlessly integrated</h2>
           </div>
           <div class="p-logo-section__item">
             {{ image(url="https://assets.ubuntu.com/v1/0ee7c9ab-nvidia-triton-server-logo.png",
-                        alt="nvidia triton server",
+                        alt="Nvidia Triton Server",
```

Comment:
FWIW, and I'm really not at all sure how much this matters, but they've requested we stylize every instance of their name on our sites using all caps. Either way it'll get read the same way by a screen reader, but if the image fails to load it'll be visible. Is it likely? Not really. But will it cause a larger issue? Also no. Why am I even leaving this comment? Not sure anymore, but felt like relevant info to share 😅

Reply:
I enjoyed reading every word hahaha but that makes sense, it is good to cover every edge case